ddtrace/contrib/grpc/__init__.py

```diff
@@ -0,0 +1,9 @@
+from ...utils.importlib import require_modules
```

Comment:
We will want a comment here explaining what this integration is, how to use it, etc.

Example: https://github.com/DataDog/dd-trace-py/blob/87b9ff74509fab917c7341f34ffabbbf14b74578/ddtrace/contrib/vertica/__init__.py#L1-L40